assets/RNAseq_report.Rmd

```diff
@@ -407,6 +1208,7 @@ The table for download can be found [here](./differential_gene_expression/final_
 Additional tables for all individual contrasts are provided [here](./differential_gene_expression/DE_genes_tables).
 
 <!-- Show table here removed as I dont like showing only top 20 genes.
+TODO Is this still necessary?
```

Comment:
Have you solved this? If the table is not shown, then text line 1202 should be removed